build-tools/packages/build-cli/src/library/dangerfile.cts

```diff
@@ -34,7 +34,7 @@ declare const danger: {
 const adoConstants = {
 	orgUrl: "https://dev.azure.com/fluidframework",
 	projectName: "public",
-	ciBuildDefinitionId: 48,
+	ciBuildDefinitionId: 11, // 11 = Build - client packages ; this definition only exists in the public ADO project
```

Comment:
Since the pipelines install build-tools from the repo by default (over the npm version), I think this change becomes effective immediately, we don't need to release build-tools and consume it first.

Reply:
What's 48? Is it used to build other pipeline and now used to build client package pipeline only?

Reply:
`ciBuildDefinitionId` defines which pipeline in ADO is generating the baseline measurements for bundle size and test coverage. The logic in the dangerfile then tries to download artifacts from runs of that pipeline to get the baselines. 48 is the pipeline definitionId for [Build - Client bundle size artifacts](https://dev.azure.com/fluidframework/public/_build?definitionId=48) (see the `definitionId=48` in the URL when you click that link), the one we used today to generate the baselines. `definitionId=11` corresponds to [Build - client packages](https://dev.azure.com/fluidframework/public/_build?definitionId=11) (in the public project! In internal it's different).